Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
> LaTeX is heavy to install so people tend not to contribute

It could be argued that (at least for these books), you can't really contribute in Pillar without installing LaTeX anyway :) 

(The issue is fresh in my mind cause I'm having various version issues at the moment and having to upgrade to LiveTeX 2015 :) ).

Since the PDF is the main product (along with the HTML), it's not very useful for people to contribute to the book in any significant way (aside from a minor spelling fix) without checking if their changes broke the generated PDF.

So, the difficulty of installation is unfortunately not removed with Pillar, but the main benefit of not having to know LaTeX syntax is pretty significant.